src/schemas/app-manifest/app/app.json

```diff
@@ -32,26 +20,14 @@
       "description": "app.runtime: FCNI to execute app, launch params.",
       "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/runtime"
     },
-    "web": {
-      "description": "app.web: Web app info; for runtime.fnci=web.",
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/web"
-    },
-    "legacy": {
-      "description": "app.legacy: Legacy app info.",
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/legacy"
-    },
-    "locations": {
-      "description": "app.locations: Allowed, excluded locations.",
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/app/locations"
-    },
     "signing": {
```

Comment:
you don't want to remove signing?

Reply:
Not sure, tbh. Figured I'd leave it in until we have a conversation about it. If you feel strongly, I can remove and we can always add it back.